### ISSUE TITLE
Doesn't work with OpenSIPS

### ISSUE THREAD

**Issue by hazratgs** [onsip/SIP.js#1067]
Hello! Recently I moved from jssip to sip.js, the move went without problems, everything works fine, but when we enabled OpenSIPS, sip.js does not work stably, every other time the call drops off with the error:

**SIP/2.0 403 Relay Forbidden**

![image](https://github.com/onsip/SIP.js/assets/4028580/c12baabf-df9a-4e97-96b0-0e01a85ba779)

On the OpenSIPS side this is the log:

```
ERROR:core:_unescape_user: invalid escaped character <4294967248>
Feb 12 08:06:56 [20] ERROR:core:compare_uris: Failed to unescape user!
```

However if I return jssip then there is no such error...

**The question is, what is this connected with? how to fix this?**

**Comment by jalsot** [comment]
> Also this project is no longer maintained so I would recommend sticking with jssip.
@seanbright I'm surprised, is it somewhere communicated what is the future of sip.js, what happened, etc? I might miss an important info...

**Comment by seanbright** [comment]
> @seanbright I'm surprised, is it somewhere communicated what is the future of sip.js, what happened, etc? I might miss an important info...

No. Just basing it on lack of commits, number of issues not being handled, etc.

**Comment by etamme** [comment]
@jaisot and @seanbright OnSIP, the company who basically sponsored the development of SIP.js was bought and sold several times, now owned by Ooma.  The original developers are still around, but there is effectively no longer any sponsored development.

**Comment by jalsot** [comment]
@seanbright @etamme Thank you both for the valuable info, I missed those happenings. 
It would be great to hear the future plans/vision from Ooma or original developers, whether they plan to continue this path or abandon the project.
